docs/stability.md

```diff
@@ -30,6 +30,7 @@ Things considered unstable for 2.x:
 * HTML generated by the web UI
 * The metrics in the /metrics endpoint of Prometheus itself
 * Exact on-disk format. Potential changes however, will be forward compatible and transparently handled by Prometheus
+* The format of the logs
```

Comment:
This being a new feature, do we really want to commit to this immediately?

Reply:
It is in the list of things that we consider unstable and don't commit to

Reply:
More particularly the list of things we'll never commit to

Reply:
My mistake sorry.